style50/core.py

```diff
@@ -110,6 +110,7 @@ def run_diff(self):
             # Display results.
             if results.diffs:
                 print(*self.diff(results.original, results.styled), sep="",  end="")
+                print()
```

Comment:
Sorry about that, I was mistaken about what the source of the issue was. The latest commit addresses it. 

The real issue is that the IDE doesn't save files with a newline at the end (though most other editors do). This is addressed in the latest commit.